client/http/request_info.go

```diff
@@ -39,6 +39,7 @@ const (
 	getStoresName                           = "GetStores"
 	getStoreName                            = "GetStore"
 	setStoreLabelsName                      = "SetStoreLabels"
+	getHealthName                           = "GetHealth"
```

Comment:
This better be the same as the method naming.

```suggestion
	getHealthName                           = "GetHealthStatus"
```